src/templates/ui-grid/ui-grid-header.html

```diff
@@ -25,4 +25,18 @@
       </div>
     </div>
   </div>
+  <div ng-if="typeof(grid.options.showTopFooter) != 'undefined' && grid.options.showTopFooter"
```

Comment:
The first half of this expression should be unnecessary.  ngIf checks if the expression is truthy.  Therefore, if grid.options.showTopFooter is undefined, null, "false", etc. it won't display.